desktopApp/electronApp/src/main.ts

```diff
@@ -45,6 +46,11 @@ async function createWindow(): Promise<void> {
     startEdgeFederatedClient(config.edgeClientConfig)
   }
 
+  mainWindow.webContents.setWindowOpenHandler(({ url }) => {
```

Comment:
target="_blank" didn't work

Reply:
It opens a new window for me on both Windows and Mac. I think its confusing because it opens a new window in the exact same spot as the original window.

Reply:
Yup, I edited my previous comment with a similar observation

Reply:
Another thing of note, standard Markdown doesn't currently have a target parameter for links. So if we want to include outside links in Computation notes we may want to modify the above code to detect links that contain "http" and "https" and then open a new window.

Reply:
good catch with markdown links.
We can solve all these cases with a link interceptor at the root of the app

Ex:
```
import React from 'react';
import ReactDOM from 'react-dom/client';
import App from './App';
import { setupLinkInterceptor } from './linkInterceptor';

// Initialize the link interceptor
const cleanupLinkInterceptor = setupLinkInterceptor();

// Render your app as usual
const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
root.render(
    <React.StrictMode>
        <App />
    </React.StrictMode>
);

```

```
// src/linkInterceptor.ts
export function setupLinkInterceptor() {
    const handleLinkClick = (event: MouseEvent) => {
        const target = (event.target as HTMLElement).closest('a');
        if (target && target.href) {
            event.preventDefault(); // Prevent default navigation

            // Open the link in a new window
            window.open(
                target.href,
                '_blank',
                'width=800,height=600,left=100,top=100' // Customize as needed
            );
        }
    };

    // Attach the event listener to the document
    document.addEventListener('click', handleLinkClick);

    // Return a cleanup function
    return () => document.removeEventListener('click', handleLinkClick);
}
```

Reply:
Ok, I've removed all changes in main.ts and have added the above code.